utils/utils.js

```diff
@@ -155,3 +184,31 @@ exports.getUnchangedDocs = (docs) => {
   return docs.filter(doc => doc.fullDescription && doc.fullDescription.indexOf('https://docs.readme.build/docs/writing-documentation') >= 0)
   .map(doc => doc.name);
 };
+
+// Parses response to make sure its the correct type
+exports.parseResponse = (response) => {
+  let parsedResponse = response.body;
+  try {
+    parsedResponse = JSON.parse(response.body, (k, v) => {
+      return v && v.type === 'Buffer' ? Buffer.from(v.data) : v;
```

Comment:
Is this the same as https://github.com/readmeio/api/pull/23/files#diff-d6c8902e1266636417b54018791967d1R34?

Reply:
They do slightly different things, but I think I should combine them